src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1647,7 +1665,16 @@ bool entitiesQuery
     }
     else if (sco->type == SCOPE_TYPE_SIMPLE_QUERY)
     {
-      qStringFilters(sco->value, filters);
+      if (qStringFilters(sco->value, filters) != true)
```

Comment:
Would be useful to add a `FIXME #1705` above this line to explain that this checking will be removed once that issue will be addressed?